engineyard/ey-cookbooks-dev-v7
# Fbz 10214 custom recipes framework

Description of your patch
-------------
Adds relevant code/structure to core recipes to be able to execute custom recipes.
Adds  a 'hello world' example to illustrate how to use custom recipes.

Recommended Release Notes
-------------
Adds relevant code/structure to core recipes to be able to execute custom recipes.
Adds  a 'hello world' example to illustrate how to use custom recipes.

Estimated risk
-------------
Low.  As is the code is not executed unless the ey-custom recipe is overwritten following the provided instructions.

Components involved
-------------
Core recipes: ey-init, to hook the execution of custom recipes.
Custom recipes: basic structure.

Dependencies
-------------
None.

Description of testing done
-------------
Enabled ey-custom and hello-world recipes.  Uploaded and apply them on an existing v7 environment.

QA Instructions
-------------
On a v7 environment where core Chef recipes run without issues:
- Follow steps as per cookbooks/ey-custom/README.md.
- Enable 'hello world' by following the steps at custom-cookbooks/hello_world/README.md.
- Upload and apply recipes on the environment, as directed by the aforementioned readme files.
- Log into the instances and check for 'Hello World" on `/var/log/chef.log`.
